docs/development.rst

```diff
@@ -0,0 +1,62 @@
+Versioning
+----------
+
+We use `semantic versioning <http://semver.org>`_. Version numbers will
+follow this format::
+
+    {Major version}.{Minor version}.{Revision number}.{Build number (optional)}
+
+Patch version numbers (0.0.x) are used for changes that are API compatible. You
+should be able to upgrade between minor point releases without any other code
+changes.
+
+Minor version numbers (0.x.0) may include API changes, in line with the
+:ref:`deprecation-policy`. You should read the release notes carefully before
+upgrading between minor point releases.
+
+Major version numbers (x.0.0) are reserved for substantial project milestones.
+
+.. _release-process:
+
+Release Process
+---------------
+
+When a new version is to be cut from the commits made on the ``develop``
+branch, the following process should be followed. This is meant to be done by
+project maintainers, who have push access to the parent repository.
+
+#. Create a branch off of the ``develop`` branch called ``release-vX.Y.Z``
+   where ``vX.Y.Z`` is the version you are releasing
+#. Use ``bump.sh`` to update the version in ``pynsot/version.py`` and the
+   Dockerfile. Example:
+
+.. code-block:: bash
+
+    $ ./bump.sh -v X.Y.Z
```

Comment:
Oops. I approved this, but there isn't a `bump.sh` for pynsot. We need to either implement one, or adjust this section to just *manually* update `pynsot/version.py`. But otherwise, good go to!